app/core/components/ContributorPathReport/index.tsx

```diff
@@ -4,149 +4,287 @@ import CheckBoxSharpIcon from "@mui/icons-material/CheckBoxSharp";
 import CheckBoxOutlineBlankSharpIcon from "@mui/icons-material/CheckBoxOutlineBlankSharp";
 import EditSharp from "@mui/icons-material/EditSharp";
 
-import {
-  CompleteIcon,
-  IncompleteIcon,
-  TipBubble,
-  HtmlTooltip,
-} from "./ContributorPathReport.styles";
+import { CompleteIcon, IncompleteIcon } from "./ContributorPathReport.styles";
 
 import type {
   ContributorPath,
   ProjectMember,
   ProjectTask,
   Stage,
 } from "~/core/interfaces/ContributorPathReport";
-import { Grid, IconButton } from "@mui/material";
+import {
+  Box,
+  Grid,
+  IconButton,
+  Link,
+  ListItem,
+  Paper,
+  Stack,
+  Table,
+  TableBody,
+  TableCell,
+  TableContainer,
+  TableFooter,
+  TableHead,
+  TablePagination,
```

Comment:
Also, can we disable pagination for this table? Since I don't think we would need it on any project (I don't see projects having more than hundreds of members 😅).

Reply:
sure!